app/mltraining_client.go

```diff
@@ -34,7 +35,7 @@ const (
 type ErrorStatus struct {
 	Code    int
 	Message string
-	Details []*GenericProtoMessage
+	Details []interface{}
```

Comment:
(q) I think I'm slightly confused about this. Can it really be _anything_ that goes into the details?

Reply:
There doesn't seem to be much clarification on what `Details` really is. The other SDKs leave it at the `Any` proto type. Converting `Any` types to `interface{}` is also something I see elsewhere in the RDK.

The only information I really see is:
```
	// A list of messages that carry the error details.  There is a common set of
	// message types for APIs to use.
```